packages/neon/neon/lib/models.dart

```diff
@@ -1,3 +1,3 @@
-export 'package:neon/src/models/account.dart';
+export 'package:neon/src/models/account.dart' hide Credentials, LoginQrcode;
```

Comment:
Just mark them as internal?

Reply:
They already are :)

But AFAIK the annotation is not hiding it and just produces a lint so we notice if it's accidentally exported.
https://dart.dev/tools/diagnostic-messages#invalid_export_of_internal_element

Reply:
I see, but is it really a problem if someone absolutely wants to import it? I mean there is probably no good reason, but still

Reply:
We'd need to provide api stability for it :thinking: 

But we can export it if you want.

Reply:
Let's just keep it and if any one ever needs it they can export it